evo_researcher/benchmark/agents.py

```diff
@@ -180,6 +195,20 @@ def predict(self, market_question: str) -> Prediction:
         except ValueError as e:
             print(f"Error in EvoAgent's predict: {e}")
             return Prediction()
+        
+    def predict_from_research(
```

Comment:
Seems like this could be used in `def predict`, to not duplicate the code around `_make_prediction`